config/routes.rb

```diff
@@ -11,4 +11,7 @@
   get "/account", to: "users#show"
   get "/login", to: "sessions#new"
   post "/login", to: "sessions#create"
+  delete "/logout", to: "sessions#destroy"
```

Comment:
Remember, DELETE is not an HTTP protocol, only GET and POST are. This needs to be GET, and the controller will handle the logic for delete.